deepmd/pt/utils/stat.py

```diff
@@ -240,6 +240,7 @@ def compute_output_stats(
     rcond: Optional[float] = None,
     preset_bias: Optional[Dict[str, List[Optional[torch.Tensor]]]] = None,
     model_forward: Optional[Callable[..., torch.Tensor]] = None,
+    intensive: Optional[bool] = False,
```

Comment:
What is the behavior of `None`? If you don't implement it, please don't add `Optional`.

Reply:
I have deleted `Optional`